### ISSUE TITLE
It's really difficult to reproduce the results of the paper

### ISSUE THREAD

**Issue by wgh-2018** [neufieldrobotics/NeuFlow#4]
I tried to use your source code for training. 
I first trained on the FlyChairs  for 100,000 steps with an initial learning rate of 4e-4. 
Then I used the pre-trained weights of FlyChairs to train the FlyThings3d dataset for 200,000 steps with an initial learning rate of 2e-4. 
The final training results were things_clean_epe:5.58, sintel_final_epe:3.81
which are significantly different from those in your paper. 
Could you please provide your training details? Thank you.
